src/screens/TestCases/Modal/CustomFilterModal.js

```diff
@@ -55,7 +106,7 @@ const CustomFilterModal = ({
       style={{ top: position.top, left: position.left }}
     >
       <div>
-        <b className="fs-6 text-custom">Submodule</b>
+        <b className="fs-6 text-custom">{columnName}</b>
```

Comment:
what is test-custom class, give proper name

Reply:
resolved